md_api/src/main/java/com/wipro/ats/bdre/md/api/InitDeploy.java

```diff
@@ -41,6 +35,10 @@ public InitDeploy() {
 
     @Autowired
     DeployDAO deployDAO;
+    public InitDeploy() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
 
     public InitDeployInfo execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)